include/object_invoke.h

```diff
@@ -77,6 +81,9 @@ namespace mil {
         constexpr void operator()(object_t & aObject, acceptor_t & aAcceptor) const {
             (*m_invokerPtr)(aObject, m_tag, aAcceptor);
         }
+        constexpr void operator()(object_t* aObject, acceptor_t & aAcceptor) const {
```

Comment:
Why don't just call the 1st overload?
Like
```cpp
constexpr void operator()(object_t * aObject, acceptor_t & aAcceptor) const {
    assert(aObject && "No object");
    (*this)(*aObject, aAcceptor);
}
```

Reply:
Some time we get ref, some time get ref to pointer.  So overload is simplest solve problem. In another cases necessary change code in many places as i think. maybe i'm wrong, but i have question. You try checkout last version and use your suggestion? Code complile and work?

Reply:
Hello sorry for this delay:)

The ref to a pointer has another meaning, ref to a pointer means that you'd like to "sync" changes to the address stored inside the pointer inside and outside the function/unit.

Here we're interesing only in accessing object's method.
And having an overload, which accepts pointer, means that we MUST think of it, since pointers may be NULL. So you MUST write "if (p != nullptr)" every time to avoid UB.

From these points it's safer just to not have public API, which accepts pointer at all. 
Just use a reference.